index.js

```diff
@@ -202,7 +202,7 @@ async function runHelmfile(projects) {
 
   const filesHaveChanged = changesToHelmfile.some(({ fileHasChanged }) => fileHasChanged)
   if (filesHaveChanged) {
-    await closePRs();
+    // await closePRs();
```

Comment:
Delete these two lines rather than comment them out?

Reply:
Just for you ❤️